test/extended/testdata/imagestream-jenkins-slave-pods.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  labels:
+    role: jenkins-slave
```

Comment:
testing both behaviors for both imagestreams and imagestreamtags seemed redundant to me (same code path utlimately)... the imagestream test does not use it, the imagestreamtag test does

that ok?